call-home/src/common/constants.rs

```diff
@@ -1 +1,7 @@
 pub(crate) const PRODUCT: &str = "Mayastor";
+pub(crate) const DATA_DIR_PATH: &str = "/encryption_dir";
+pub(crate) const ENCRYPTION_KEY_FILEPATH: &str = "/key/";
+pub(crate) const SWARM_API_ENDPOINT: &str = "";
+pub(crate) const SWARM_CA_CERT_FILEPATH: &str = "/ca_cert/Caringo_Root_CA_2015.pem";
```

Comment:
This should be removed, along with the certificate itself.

Reply:
Done.